sky/global_user_state.py

```diff
@@ -56,7 +56,13 @@ def create_table(cursor, conn):
         handle BLOB,
         last_use TEXT,
         status TEXT,
-        autostop INTEGER DEFAULT -1)""")
+        autostop INTEGER DEFAULT -1,
+        metadata TEXT DEFAULT "{}",
+        to_down INTEGER DEFAULT 0,
+        owner TEXT DEFAULT null,
+        cluster_hash TEXT DEFAULT null,
+        storage_mounts_metadata BLOB DEFAULT null,
+        cluster_ever_up INTEGER DEFAULT 0)""")
```

Comment:
No. It will be the same as before, but this can be more efficient for new users as we don't need to add new cloumns multiple times for the new users. Also, we can gradually remove those `add_column_to_table()` function.